go/filemanager/pfsmod/chunk.go

```diff
@@ -0,0 +1,118 @@
+package pfsmod
+
+import (
+	"errors"
+	"fmt"
+	log "github.com/golang/glog"
+	"io"
+	"net/url"
+	"os"
+	"strconv"
+)
+
+//ChunkCmd structure
```

Comment:
The code next line already shows it's a struct, I think we need a more descriptive comment.
Also, please leave a space after "//": `// comment`.
Please see: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences